h2o-docs/src/booklets/v2_2015/source/sw/sections/usecase.tex

```diff
@@ -25,7 +25,7 @@ \subsection{Predicting Arrival Delay in Minutes - Regression}
     val conf: SparkConf = configure("Sparkling Water: Join of Airlines with Weather Data")
 
     // Create SparkContext to execute application on Spark cluster
-    val sc = new SparkContext(conf)
+    val spark = SparkSession.builder().config(conf).appName("SparkSessionZipsExample").getOrCreate()
```

Comment:
last thing, sorry for bothering that much. 

the name of app is defined on the line above ( I didn't notice it at the start ) so this line should be just

`val spark =SparkSession.builder().config(conf).getOrCreate()`